# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../acx100/linux-2.6.16.patch
# Copyright (C) 2006 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./usb.c.orig	2006-10-07 07:09:19.000000000 -0400
+++ ./usb.c	2006-10-07 09:33:24.000000000 -0400
@@ -156,6 +156,9 @@
 static struct usb_driver
 acxusb_driver = {
 	.name = "acx_usb",
+	.driver = {
+	.owner = THIS_MODULE,
+	},
 	.probe = acxusb_e_probe,
 	.disconnect = acxusb_e_disconnect,
 	.id_table = acxusb_ids
